src/NzbDrone.Core/Indexers/Definitions/Xthor/XthorParser.cs

```diff
@@ -18,7 +18,7 @@ public XthorParser(XthorSettings settings, IndexerCapabilitiesCategories categor
         {
             _settings = settings;
             _categories = categories;
-            _torrentDetailsUrl = _settings.BaseUrl.Replace("api.", "") + "details.php?id={id}";
+            _torrentDetailsUrl = _settings.BaseUrl.Replace("api.", "") + "/details.php?id={id}";
```

Comment:
```suggestion
            _torrentDetailsUrl = _settings.BaseUrl.Replace("api.", "").TrimEnd('/') + "/details.php?id={id}";
```

Add trimend to this to ensure it doesn't double in default case. This could probably be cleaned up more as this is only used iin one method, so doesn't really need to be a field defined on class